_posts/2018-11-29-blog-build-chrome-ringcentral-widgets-extension-for-crms.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Build Chrome RingCentral widgets extension for CRMs"
+date:   2018-11-29 13:31:07 +0800
+categories: blog
+---
+
+## Why is it need?
+
+- First, and the important reason, I guess, is Our clients need it, so for big CRM, like salesforce, we even build special app for it. For not that big CRMs, still have clients need it too, we may not build a special app for it. But we have the powerful widgets, so we can build a chrome extension easily.
+- Second, our RingCentral widgets is userful and powerful for these CRMs, we could greatly improve their work proccess.
+
+## What will we do in these extensions?
+
+In general, All the third party features of our [ringcentral-embeddable](https://github.com/ringcentral/ringcentral-embeddable) supported and some content insert:
+
+- For CRM contact list, we will add a hover-to-show tooltip to show click-to-call button.
+
+![ ](https://github.com/zxdong262/ringcentral-embeddable-extension-factory/raw/master/screenshots/fac-2.png)
+
+- For CRM contact info page, we will add a click-to-call button in proper position.
+
+![ ](https://github.com/zxdong262/ringcentral-embeddable-extension-factory/raw/master/screenshots/fac-3.png)
+
+- For CRM contact phone number text, we make it click-to-call link.
+
+![ ](https://github.com/zxdong262/ringcentral-embeddable-extension-factory/raw/master/screenshots/fac-1.png)
+
+- Sync CRM contacts to our widgets after user authorize.
+
+![ ](https://github.com/zxdong262/insightly-embeddable-ringcentral-phone/raw/master/screenshots/insightly-4.png)
+
+- Sync call log to CRM automatticly or mannually.
+
+![ ](https://github.com/zxdong262/hubspot-embeddable-ringcentral-phone/raw/master/screenshots/hs6.png)
+
+- Check CRM conatct activities from our widgets.
+
+![ ](https://github.com/zxdong262/hubspot-embeddable-ringcentral-phone/raw/master/screenshots/hs7.png)
+
+- Show CRM contact info panel when inbound/outbpund call with CRM contact.
+
+![ ](https://github.com/zxdong262/hubspot-embeddable-ringcentral-phone/raw/master/screenshots/hubspot1.png)
+
+## How do we do it
+
+Ok, two steps, build a chrome extension, and add all the above features, with [Embbnux Ji](https://github.com/embbnux)'s tuturial:
```

Comment:
Again, if you are listing steps in a formal writing. Just use a list:
1. We will do ... .
2. We will do ... . 
3. ...